mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -232,9 +232,10 @@
 
 	See: https://github.com/WordPress/wporg-mu-plugins/issues/159
 	*/
-
-	& .wp-block-navigation__container {
-		background: var(--wp--preset--color--dark-grey);
+	@media (--tablet) {
+		& .wp-block-navigation__container {
+			background: var(--wp--preset--color--dark-grey);
+		}
```

Comment:
I like to put the media query inside the element. That way default mobile styles can go in there too without having to create a duplicate block. eg.,

```postcss
& .wp-block-navigation__container {
	/* other stuff can go here in the future */

	@media (--tablet) {
		background: var(--wp--preset--color--dark-grey);
	}
}
```